package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A go style channel implementation that works nicely with co",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "test": "babel-node --stage=0 node_modules/.bin/blue-tape src/**/*.test.js"
```

Comment:
I was on a kick where I really liked `tape` when I wrote this, but I am good with sticking with `mocha`